packages/simm-helpers/format-utils.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * Convert phone number to phone number with hyphens.
```

Comment:
`Convert phone number to phone number`  cái gì z em @@..., với lại cái này dùng regex chứ k dùng cách như hiện tại nhé.